auth/tests/unit/attestation.unit.test.ts

```diff
@@ -1,67 +1,78 @@
-import { describe, it } from "vitest";
-import { loadTemplateFromFile } from '../common/template'
+import { beforeAll, describe, it } from "vitest";
+import { loadTemplateFromFile } from "../common/template";
 
-const template = loadTemplateFromFile('./template.yaml')
+const template = loadTemplateFromFile("./template.yaml");
 
 describe("attestation", () => {
+  let resourceUnderTest: {
+    Type: any;
+    Properties: any;
+  };
 
-    describe("provisions an auth proxy lambda handler", () => {
-        it('should resolve attestation app ids as environment variables', () => {
-            template.hasResourceProperties("AWS::Serverless::Function", {
-                Environment: {
-                    Variables: {
-                        FIREBASE_ANDROID_APP_ID: "{{resolve:ssm:/firebase/appcheck/android-app-id}}",
-                        FIREBASE_IOS_APP_ID: "{{resolve:ssm:/firebase/appcheck/ios-app-id}}"
-                    }
-                }
-            })
-        })
-    })
+  beforeAll(() => {
+    const resource = template.findResources("AWS::Serverless::Function");
+    resourceUnderTest = resource["AuthProxyFunction"] as any; // find Post Authentication Lambda function
```

Comment:
Nice! Much more concise

```suggestion
    resourceUnderTest = resource["AuthProxyFunction"] as any; 
```